test/integration/targets/ansible-galaxy-collection/tasks/unsupported_resolvelib.yml

```diff
@@ -0,0 +1,43 @@
+- name: get installed resolvelib version
+  shell: "{{ ansible_python_interpreter }} -m pip freeze | grep 'resolvelib'"
+  register: installed_resolvelib
+
+- block:
```

Comment:
I think this test should be moved to its own target and use a runme.sh to setup a venv. Otherwise we risk overwriting an OS package install of resolvelib with pip.

Reply:
Good catch, fixed. Now the unsupported resolvelib versions are tested using a venv. I also fixed import errors for very outdated (or missing) resolvelib.